allDir.rb

```diff
@@ -0,0 +1,3 @@
+Dir.chdir(".")
+subdir_list=Dir["*"].reject{|o| not File.directory?(o)}
+puts subdir_list;
```

Comment:
Wrap it in a method and call that method. Always try to think if your code will then new functionality should not break the old one. It should be backward compatible.